third_party/pyth/price-service/src/__tests__/rest.test.ts

```diff
@@ -146,3 +156,17 @@ describe("Latest Vaa Bytes Endpoint", () => {
     expect(resp.body.message).toContain(ids[2]);
   });
 });
+
+// describe("Get Vaa Endpoint", () => {
```

Comment:
Are you going to add a test later? It's better to have it fully tested on this PR, but if not, i suggest removing this comment block and add it later when you add the tests.